src/main.java

```diff
@@ -1,2 +1,7 @@
 public class main {
```

Comment:
```suggestion
public class Main {
```

Java에서는 보통 클래스 이름의 앞문자를 대문자로 써줍니다.